Gruntfile.js

```diff
@@ -66,6 +66,7 @@ const distAssets = `${distDir}/assets`
 const distCSS = `${distDir}/assets/css`
 const distContracts = `${distDir}/contracts`
 const distJS = `${distDir}/assets/js`
+const distBackend = `${distDir}/backend`
```

Comment:
Is this building the Chelonia Dashboard? If so, could you add a comment either here or below stating that? If not, could you make sure that the Chelonia Dashboard is also built?

BTW, PR #2783 updating the Chelonia dashboard was just merged into `master` so please merge `master`